src/routes/articles/get-article.e2e.test.js

```diff
@@ -0,0 +1,95 @@
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+
+
+describe('GET /articles/:id', () => {
+    
+	let user
+	let accessToken
+
+	before(async ()=>{
+		 user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+	})
+
+	describe('Failure', () => {
+
+		it('should return 400 if article.id is invalid', async () => {
+			const expectedError = {
+				"status" : "failed",
+				"error": {
+			  "message": "id must be a number"
+				}
+		  }
+			return	fixtures.api()
+				.get(`/api/v1/articles/sdsdsd`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(400, expectedError)
+
+		})
+		
+	})
```

Comment:
kindly add a test for article not existing

Reply:
yes please